pages/index.tsx

```diff
@@ -1,19 +1,24 @@
 import { useSession, signIn, signOut } from "next-auth/client";
+import { Navbar } from "@components/Navbar";
+import { Footer } from "@components/Footer";
 
 export default function Component() {
     const [session, loading] = useSession();
     if (session) {
         return (
             <>
+                <Navbar session={session} />
                 Signed in as {session.user.email} <br />
                 <button onClick={() => signOut()}>Sign out</button>
+                <Footer />
```

Comment:
![image](https://user-images.githubusercontent.com/46411197/126245593-0bc4a62b-7baf-4ec0-9eeb-2b0f66e37733.png)

Can the footer be moved to the bottom?

Reply:
It technically is at the bottom, there just isn't enough content to fill the page which is why it looks like it's in the middle. I think it would be up to the parent component to just ensure that the minimum height of the container is 100% of the screen size so that the footer touches the bottom of the screen